src/parser/expression.js

```diff
@@ -290,6 +290,13 @@ export default class ExpressionParser extends LValParser {
 
         const startPos = this.state.start;
         const startLoc = this.state.startLoc;
+
+        if (node.operator === "|>") {
+          this.expectPlugin("pipelineOperator");
+          // Support syntax such as 10 |> x => x + 1
+          this.state.potentialArrowAt = startPos;
```

Comment:
I don't think the current draft spec has this logic. I couldn't figure out how to put it into the grammar. Right now, `x |> y => z` would be a syntax error. But it's clearly useful--i'll think more about the spec.

Reply:
Good to know